discord/guild.py

```diff
@@ -290,6 +303,8 @@ def _from_data(self, guild):
         self.preferred_locale = guild.get('preferred_locale')
         self.discovery_splash = guild.get('discovery_splash')
         self._rules_channel_id = utils._get_as_snowflake(guild, 'rules_channel_id')
+        self.approximate_presence_count = data.get('approximate_presence_count')
```

Comment:
same with member count
```suggestion
        self.approximate_presence_count = guild.get('approximate_presence_count')
```